apps/frontend/src/components/xpress/XpressProposalFilterBar.tsx

```diff
@@ -0,0 +1,140 @@
+import Grid from '@mui/material/Grid';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import CallFilter from 'components/common/proposalFilters/CallFilter';
+import DateFilter from 'components/common/proposalFilters/DateFilter';
+import InstrumentFilter from 'components/common/proposalFilters/InstrumentFilter';
+import ProposalStatusFilter from 'components/common/proposalFilters/ProposalStatusFilter';
+import TechniqueFilter from 'components/common/proposalFilters/TechniqueFilter';
+import {
+  Call,
+  InstrumentFragment,
+  ProposalsFilter,
+  ProposalStatus,
+  TechniqueFragment,
+} from 'generated/sdk';
+
+type ProposalFilterBarProps = {
+  calls?: { data: Call[]; isLoading: boolean };
+  instruments?: { data: InstrumentFragment[]; isLoading: boolean };
+  techniques?: { data: TechniqueFragment[]; isLoading: boolean };
```

Comment:
Do not techniques have related instruments .

Reply:
Yes, techniques have related instruments. We pass only the instruments that are part of the techniques, to be listed in filter dropdown.